.licenserc.yaml

```diff
@@ -0,0 +1,25 @@
+header:
+  license:
+    spdx-id: Apache-2.0
+    copyright-owner: Canonical Ltd.
+    content: |
+      Copyright [year] [owner]
+      See LICENSE file for licensing details.
+  paths:
+    - '**'
+  paths-ignore:
```

Comment:
Aren't all of these ignored by default?

Reply:
`lib/**` and `.woke.yaml` aren't ignored by default. Maybe I should update the default in `operator-workflows`

Reply:
Oh, right, `lib`. I left that one out intentionally so that our own libs are checked